test/system/books_test.rb

```diff
@@ -4,42 +4,60 @@
 
 class BooksTest < ApplicationSystemTestCase
   setup do
-    @book = books(:one)
+    @book = books(:harry_potter_book)
+
+    visit root_url
+    fill_in 'Eメール', with: 'alice@example.com'
+    fill_in 'パスワード', with: 'password'
+    click_button 'ログイン'
+    assert_text 'ログインしました。'
   end
 
   test 'visiting the index' do
     visit books_url
-    assert_selector 'h1', text: 'Books'
+    assert_selector 'h1', text: '本の一覧'
+    assert_text 'ハリー・ポッター'
+    assert_text '1990年代のイギリスを舞台に、魔法使いの少年ハリー・ポッターの学校生活や、強大な闇の魔法使いヴォルデモートとの因縁と戦いを描いた物語。'
+    assert_text 'J・K・ローリング'
```

Comment:
```suggestion
    assert_text 'ハリー・ポッター''
```

「一覧に本が表示されている」ということを検証するだけであればタイトルの表示を確認するだけでも十分かなーと思います。

Reply:
タイトルのみの確認に変更しました!